pom.xml

```diff
@@ -35,7 +37,7 @@
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-java</artifactId>
-            <version>${selenium.java.version}</version>
+            <version>4.0.0</version>
```

Comment:
can't get Selenide's latest version 6.0.3 to work with Selenium lower than 4.0.0. 
Maybe it's better to remove Selenium dependency since we are not using it anymore?

Reply:
I think it better to remove selenium, it's better to do it after all usage disappears from the code